### PR TITLE
DAOSGCP-85 Add daos_client_install_script output to daos_server module

### PR DESCRIPTION
**Pull request by markaolson**
Needed for HPC toolkit

Signed-off-by: Mark A. Olson <mark.a.olson@intel.com>

### CODE REVIEW

File: terraform/examples/daos_cluster/variables.tf [hunk 1]
```diff
@@ -148,6 +148,7 @@ variable "server_daos_scm_size" {
 variable "server_daos_crt_timeout" {
   description = "crt_timeout"
   default     = 300
+  type        = number
```
Comment (markaolson) [inline terraform/examples/daos_cluster/variables.tf:151]: Added this because pre-commit hook complained that this var was missing a type.

Comment (cboneti) [reply terraform/examples/daos_cluster/variables.tf:151]: yes, it is a good practice to specify types.

**Review by cboneti — APPROVED** [review]
Thank you for doing this, this is very helpful for us.

**Review by lsitkiew — APPROVED** [review]
Looks good to me.

File: terraform/modules/daos_server/scripts/daos_client_install_script.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    exit 1
+    ;;
+esac
+
+echo "Adding DAOS v${DAOS_VERSION} packages repo"
+curl -s --output /etc/yum.repos.d/daos_packages.repo "https://packages.daos.io/v${DAOS_VERSION}/${DAOS_OS_VERSION}/packages/x86_64/daos_packages.repo"
+
+echo "Installing daos-client and daos-devel packages"
+yum install -y daos-client daos-devel
+systemctl enable daos_agent
```
Comment (Kmannth) [inline terraform/modules/daos_server/scripts/daos_client_install_script.sh:37]: @markaolson   Is it safe to enable now or that should be with the installation of the YML files?  I was just working in the boot time area and there is a spot where the default YML gets uses and it is not great.

Comment (markaolson) [reply terraform/modules/daos_server/scripts/daos_client_install_script.sh:37]: It's only getting enabled at this point but it's not started yet. 
The service actually gets started after the yml files get deployed in a separate "configure" script.

Comment (Kmannth) [reply terraform/modules/daos_server/scripts/daos_client_install_script.sh:37]: ok

**Pull request merged**